Automattic/jetpack
# Adds the atomic flag to the Jetpack Blaze base site's configuration

We want to distinguish between atomic and non-atomic sites using Jetpack Blaze. For this reason, we are adding the configuration option `is_wpcom_atomic` to the site's option when we load the Blaze Dashboard app.

In parallel, we are working on a https://github.com/Automattic/wp-calypso/pull/88858, which will use that atomic flag to change our Blaze origin in track events.

## Proposed changes:
* Adds the site's option `is_wpcom_atomic` to the Blaze dashboard initial configuration

### Other information:


## Jetpack product discussion


N/A

## Does this pull request change what data or activity we track or use?


No

## Testing instructions:






* Go to Tools->Advertising
* Verify that the Jetpack Blaze dashboard is loaded correctly
